features/shadow/core/src/main/java/org/apache/shardingsphere/shadow/rule/attribute/ShadowTableMapperRuleAttribute.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.shadow.rule.attribute;
+
+import com.cedarsoftware.util.CaseInsensitiveSet;
+import org.apache.shardingsphere.infra.rule.attribute.table.TableMapperRuleAttribute;
+
+import java.util.Collection;
+import java.util.Collections;
+
+/**
+ * Shadow table mapper rule attribute.
+ */
+public class ShadowTableMapperRuleAttribute implements TableMapperRuleAttribute {
+    
+    private final CaseInsensitiveSet<String> logicalTableMapper;
+    
+    public ShadowTableMapperRuleAttribute(final Collection<String> encryptTableNames) {
```

Comment:
`encryptTableNames`  should not here

Reply:
Thank you very much for your review. It was my carelessness that caused this issue. I have adjusted the variable names and want to express my gratitude again.